controllers/grid/articleGalleys/ArticleGalleyGridHandler.php

```diff
@@ -76,7 +76,7 @@ public function getSubmission()
     /**
      * Get the authorized publication.
      *
-     * @return Publication
+     * @return \Publication
```

Comment:
Rather than add slashes here, it is better to add the use statements to the top of the file:

```
use APP\submission\Submission;
use APP\publication\Publication;
```

Reply:
Fixed